src/scancode/cli.py

```diff
@@ -406,10 +406,16 @@ def print_options(ctx, param, value):
     hidden=True,
     help_group=MISC_GROUP, sort_order=1000, cls=CommandLineOption)
 
+@click.option('--max-depth',
+    type=int, default=0, show_default=False, metavar='N',
+    help='For each <input> given on the command line, scan at most N levels '
+        'of directories. A negative value or 0 means no limit. 0 is the default',
+    help_group=CORE_GROUP, sort_order=10, cls=CommandLineOption)
+
```

Comment:
What about this?

```Python
@click.option('--max-depth',
    type=int, default=0, show_default=False, 
    help='Maximum nesting depth of subdirectories to scan. '
        'Descend at most INTEGER levels of directories below and including '
        'the starting directory. Use 0 for no scan depth limit.',
    help_group=CORE_GROUP, sort_order=10, cls=CommandLineOption)


```
Also can you move this up right after the `--max-in-memory` option, since it is part of the core group?

Reply:
Yeah sure. Will do that :)